app/views/layouts/_nav.html.erb

```diff
@@ -0,0 +1,13 @@
+<% if logged_in? %>
+  <p style="float:right;">Hi <%= current_user.username %></p>

```

Comment:
Love the style

Reply:
Yeah, really gettin' crazy with the CSS there!